app/build.gradle

```diff
@@ -36,7 +36,13 @@ android {
     }
 }
 
+repositories {
+    mavenCentral()
```

Comment:
Add this in the root `app.gradle`. Where the other `repositories` are defined.

Reply:
yeah makes sense